src/app/vulnerabilities/packageJson.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Factory\AppFactory;
+
+$app->get('/package.json', function (Request $request, Response $response) {
+    $packageData = [
+        'name' => 'Your Package Name',
+        'version' => '1.0.0',
+        'description' => 'Description of your package',
+        'author' => 'Your Name',
+        
+    ];
+
+    // Convert the PHP array to a JSON string
+    $packageJson = json_encode($packageData, JSON_PRETTY_PRINT);
+
+    // Set the JSON content type and return the package.json content
+    $response->getBody()->write($packageJson);
+    return $response->withHeader('Content-Type', 'application/json')->withStatus(200);
+});
```

Comment:
package.json must have this type of content https://github.com/heroku/node-js-sample/blob/master/package.json. Copy paste the following text & add value in depedencies key any long random name

Reply:
![Screenshot from 2023-11-08 19-06-35](https://github.com/getastra/hypejab/assets/87857234/a30f4a49-1d89-4bcf-926b-3242404b0192)

Reply:
is this fine? @divyanshjainATastra